app/controllers/items/select_code.js

```diff
@@ -23,9 +23,11 @@ export default SearchCode.extend({
     const item = this.get("item");
     const type = item.get("detailType");
     const detailId = item.get("detailId");
-    await this.runTask(
-      this.get("subformDetailService").deleteDetailType(type, detailId)
-    );
+    if (type) {
```

Comment:
@abulsayyad123 , its better to check type one of ["electrical", "computer", "computer_accessory"] instead of just checking type because some of the package_types/items have detail_type as "medical" and it might cause an issue going forward. 

:)

Reply:
This check is for delete subform if there exist subform.

Reply:
okay, makes sense 👍